tests/unit/commands/test_validation_sets.py

```diff
@@ -229,3 +232,177 @@ def test_edit_validation_sets_with_changes_to_existing_set(
             key_name=key_name,
         )
     ]
+
+
+@pytest.mark.usefixtures("memory_keyring")
+def test_edit_validation_sets_with_errors_to_amend(
+    validation_sets_payload,
+    fake_dashboard_get_validation_sets,
+    fake_dashboard_post_validation_sets_build_assertion,
+    fake_dashboard_post_validation_sets,
+    fake_snap_sign,
+    fake_edit_validation_sets,
+    monkeypatch,
+):
+    fake_edit_validation_sets.return_value = {
+        "account-id": "AccountIDXXXOfTheRequestingUserX",
```

Comment:
This one's just a suggestion: the checks that the tests do are themselves fine, but I think the amount of "hardcoded" text makes it hard to see what's actually being tested. For example, consider putting this dict in a variable that you can use both here and in the `assert fake_dashboard_post_validation_sets_build_assertion.mock_calls` check

Reply:
sounds like a good suggestion, so yes, I will do this.